src/components/badge/badge.tsx

```diff
@@ -11,12 +11,17 @@ import { abbreviate } from './format';
 })
 export class Badge {
     /**
-     * Label to display in the badge
+     * Label to display in the badge.
+     * String labels only allow up to 5 characters.
      */
     @Prop({ reflect: true })
-    public label: number;
+    public label: number | string;
 
     render() {
-        return abbreviate(this.label);
+        if (typeof this.label === 'number') {
+            return abbreviate(this.label);
+        } else if (typeof this.label === 'string') {
+            return truncate(this.label);
```

Comment:
From what I see in the preview docs, this doesn't really "truncate" the label, which is not good

We want this 👇 :
<img width="76" alt="image" src="https://user-images.githubusercontent.com/35954987/193859318-3528c331-fe7d-4cb1-b166-0d7bb385757f.png">

but we are getting this 👇 
<img width="55" alt="image" src="https://user-images.githubusercontent.com/35954987/193859487-cc7ff5db-7ed6-4026-b67e-e372069b8213.png">

Also, this creates accessibility issues, because:
1. the word is in fact turned to `Webco` in the DOM. So even screen readers will read it out loud like that
2. there is no way for a sighted user to know what the word is, for instance by hovering on the component and seeing it as a tooltip. So putting this as `title` attribute on the badge would be a great solution.

To avoid these, you can leave the truncation job to CSS. Thus we don't need a function for it at all.